app/main.py

```diff
@@ -1,3 +1,11 @@
 def cache(func):
-    # Write your code here
-    pass
+    cash = {}
+
+    def inner(*args, **kwargs):
+        nonlocal cash
+        if args not in cash:
+            print("Calculating new result")
+            cash[args] = func(*args, **kwargs)
+        else:
+            print("Getting from cache")
```

Comment:
You should return a result of the function. I have fixed test_main.py today, please update your `tests/test_main.py` from origin/master via `git checkout master -- tests/test_main.py`